packages/vue-sample/src/router/index.ts

```diff
@@ -0,0 +1,27 @@
+import { createRouter, createWebHistory } from "vue-router";
+
+import CallbackView from "../views/CallbackView.vue";
+import HomeView from "../views/HomeView.vue";
+
+const router = createRouter({
+  history: createWebHistory(),
+  routes: [
+    {
+      path: "/",
+      name: "home",
+      component: HomeView,
+    },
+    {
+      path: "/callback",
+      name: "callback",
+      component: CallbackView,
+    },
+    {
+      path: "/protected-resource",
+      name: "protected-resource",
+      component: () => import("../views/ProtectedResourceView.vue"),
```

Comment:
just curious, what make it special to use "import()"

Reply:
Oh... nothing special, just showing off you can actually lazy load this module... lol